07-modeling.Rmd

```diff
@@ -224,11 +228,11 @@ m_electric_multi <- recs_des %>%
   svyglm(
     design = .,
     formula = DOLLAREL ~ (Region + Urbanicity + TOTSQFT_EN + ACUsed) ^
-      2 - 1,
+      2 - 1, 
     na.action = na.omit
   )
 
-tidy(m_electric_multi) %>% print(n = 50)
+tidy(m_electric_multi) %>% print(n = 50) 
```

Comment:
Just a note that line 21 wraps in the PDF.  It stays on the page but the last number wraps to the next line.  Not sure if there is anything we can do about it, but just noting it.